homework1/task1.js

```diff
@@ -0,0 +1,20 @@
+import reverseString from "./reverseString";
+
+const printOutput = (data)=>{
+    process.stdout.write(data);
+    process.stdout.write("\n\n\n");
+    prompt();
+}
+
+const onReadData = (data)=>{
+    const reversedString = reverseString(data);
+    printOutput(reversedString);
+}
+
+const prompt = ()=>{
+    process.stdout.write("Enter a text!\n");
+}
+
+
+prompt();
+process.stdin.on('data',onReadData);
```

Comment:
Also handle process termination. Note the difference between with and w/o graceful termination.

[Graceful termination](https://hackernoon.com/graceful-shutdown-in-nodejs-2f8f59d1c357)

[SIGTERM vs SIGINT](https://stackoverflow.com/questions/42450501/catching-sigterm-vs-catching-sigint/48959557)

Reply:
Thanks @Ayudh, I have added the SIGTERM and SIGINT handlers. Please check.